app/src/main/java/com/sun/heartrate/utils/ViewExtension.kt

```diff
@@ -0,0 +1,11 @@
+package com.sun.heartrate.utils
+
+import android.view.View
+
+fun View.gone() {
```

Comment:
anh nên viết với những trường hợp View chưa chắc đã nonNull, 
```suggestion
fun View?.gone() {
```

Reply:
Code này đúng rồi. Khi sử dụng mới cần check null bằng dấu ?